src/Components/OATPropertyEditor/JSONEditor.tsx

```diff
@@ -20,9 +29,13 @@ const JSONEditor = ({ dispatch, theme, state }: JSONEditorProps) => {
     const editorRef = useRef(null);
     const { model } = state;
     const [content, setContent] = useState(null);
+    const unSavedContent = useRef(null);
```

Comment:
My understanding is that we're using this to hold the "original" so we can cancel the change.

Two questions:
1. Can we just re-fetch it from model on cancel? I.e. `JSON.stringify(model, null, 2);`, or
2. Can we use a state property instead (i.e. rather than a ref)?